README.md

```diff
@@ -142,6 +142,7 @@ The following [dispatchers](./telemetry/producer.go#L10-L19) are supported
   * Configure stream names directly by setting the streams config `"kinesis": { "streams": { *topic_name*: stream_name } }`
   * Override stream names with env variables: KINESIS_STREAM_\*uppercase topic\* ex.: `KINESIS_STREAM_V`
 * Google pubsub: Along with the required pubsub config (See ./test/integration/config.json for example), be sure to set the environment variable `GOOGLE_APPLICATION_CREDENTIALS`
+* ZMQ: Configure with the config.json file.  See implementation here: [config/config.go](./config/config.go)
```

Comment:
`make test` does not work without installing zmq first. Update readme for how to install for mac and linux?
Also add a step to github actions https://github.com/teslamotors/fleet-telemetry/blob/main/.github/workflows/build.yml#L18. Something like:-
```sh
jobs:
  build:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v3

    - name: Set up Go
      uses: actions/setup-go@v3
      with:
        go-version: 1.20.3

    - name: Format
      run: |
        make format
        git diff --exit-code
    - name: Install ZeroMQ
      run: |
        sudo apt-get update
        sudo apt-get install -y libzmq3-dev

    - name: Test
      run: make test

    - name: Integration Test
      run: make integration
```

Reply:
Will do, would you prefer to build from source or apt install?

Reply:
minor: if it doesn't hurt performance we should add this check and an error with the action to take to `make test`